accounting/rewind_test.go

```diff
@@ -67,10 +67,12 @@ func TestStaleTransactions1(t *testing.T) {
 		Round:   8,
 	}
 
-	var outCh <-chan idb.TxnRow
+	ch := make(chan idb.TxnRow)
+	var outCh <-chan idb.TxnRow = ch
+	close(ch)
 
 	db := &mocks.IndexerDb{}
-	db.On("GetSpecialAccounts").Return(transactions.SpecialAddresses{}, nil)
+	db.On("GetSpecialAccounts", mock.Anything).Return(transactions.SpecialAddresses{}, nil)
```

Comment:
I'm not sure why this one was working. It should have started failing once we added the context to this function.